# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/ncompress/config.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


ncompress uses a really nice build script with a few evil bugs ...

 - Clifford

--- ./build	Wed Oct 28 12:10:53 1992
+++ ./build.new	Mon Aug 31 21:25:22 1998
@@ -278,8 +278,6 @@
 		[ -f /usr/include/sys/dir.h ] && { SYSDIR=yes; }
 		[ -f /usr/include/dirent.h ] && { DIRENT=yes; }
 		[ -f /usr/include/utime.h ] && { UTIME_H=yes; }
-		[ -d /usr/local/bin ] && { BINDIR=/usr/local/bin; }
-		[ -d /usr/local/man ] && { BINDIR=/usr/local/man/man1; }
 		[ -f /usr/bin/compress ] && { BINDIR=/usr/bin; }
 	
 		if [ ".${CC}" = . ]
@@ -790,7 +788,7 @@
 
 
 compress:	Makefile compress42.c patchlevel.h
-	$(CC) -o compress $(options) "-DCOMPILE_DATE=\"\`date\`\"" compress42.c $(LBOPT)
+	\$(CC) -o compress \$(options) "-DCOMPILE_DATE=\\"\`date\`\\"" compress42.c \$(LBOPT)
 
 install:	compress
 		[ -f \$(BINDIR)/compress ] && \\
